setenv.sh

```diff
@@ -0,0 +1,19 @@
+# BITGO configuration
```

Comment:
could this script read config/config.docker.js and set the variables from that? i'd like to keep configuration information in one place (`config/`) and in the same format (`.js`)